src/aiidalab_qe/plugins/bands/__init__.py

```diff
@@ -15,8 +16,15 @@ class BandsOutline(OutlinePanel):
 """
 
 
+projwfc_code = QEAppComputationalResourcesWidget(
+    description="projwfc.x",
+    default_calc_job_plugin="quantumespresso.projwfc",
+)
+
+
 bands = {
     "outline": BandsOutline,
+    "code": {"projwfc": projwfc_code},
```

Comment:
Hi @AndresOrtegaGuerrero ,  this should fix the failed tests. Because there is already a code named `projwfc` in pdos plugin, so you need use another name. 
```suggestion
    "code": {"projwfc_bands": projwfc_code},
```
I will open another PR to handle this automatically.

Reply:
Thank you!